CMakeLists.txt

```diff
@@ -307,6 +307,9 @@ endif()
 # Catch2 for tests
 set(CATCH_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/lib/catch2)
 
+add_library(catch::catch INTERFACE IMPORTED)
+target_include_directories(catch::catch INTERFACE ${CMAKE_SOURCE_DIR}/include ${CATCH_INCLUDE_DIR})
```

Comment:
I thought there was already a target? `catch2::catch2` maybe? Their own official target last I checked is `Catch2::Catch2WithMain`, which we don't use (but maybe should for consistency).

Reply:
We vendor the catch header file so we don't use the find package.  I just wanted the prettier way of getting the include :smile:

Reply:
lib/CMakeLists.txt#L35-36

Reply:
oh, that's not in the convenient spot!